src/main/resources/META-INF/mods.toml

```diff
@@ -27,4 +27,11 @@ modId = "caffeineconfig"
 mandatory = true
 versionRange = "[1.0,)"
 ordering = "NONE"
-side = "BOTH"
\ No newline at end of file
+side = "CLIENT"
+
+[[dependencies.sodiumextra]]
+modId = "minecraft" # Gives a friendlier error message when installed on the wrong MC version
+mandatory = true
+versionRange = "[1.19.4,1.20)" # Allows 1.19.4 to 1.19.9
+ordering = "NONE"
+side = "CLIENT"
```

Comment:
The original mod is independent of the minecraft version

Reply:
If you run the 1.19.4 version of Rubidium Extras on 1.18.2, it crashes. This gives a more helpful error message saying something along the lines of "You have Minecraft 1.18.2 but (mod file name) requires 1.19.4".

The original mod will also crash if you attempt to use the wrong version for your MC.